packages/aragon-wrapper/src/index.js

```diff
@@ -935,9 +935,23 @@ export default class Aragon {
    *
    * @return {Promise<void>}
    */
+  
   clearLocalIdentities () {
     return this.identityProviderRegistrar.get('local').clear()
   }
+  
+  
+  /**
+   * Clear selected local identities
+   *
+   * @return {Promise<Object>}
```

Comment:
Let's remove this too as the function does not return anything

Reply:
cc @2color whats the team convention here? If the function is `async` it will return a `Promise` no? what do we set here?

Reply:
I'm pretty sure we just write Promise if the value doesn't matter that much, i.e. `@return {Promise}`

I'd add a @param documenting the type of selectedAddresses, e.g.
https://github.com/aragon/aragon.js/blob/fcdb8700c2d6e104b8115499e198768db38e54f9/packages/aragon-wrapper/src/index.js#L1201